gammapy/maps/axes.py

```diff
@@ -2187,7 +2199,7 @@ def format_plot_xaxis(self, ax):
         import matplotlib.pyplot as plt
         from matplotlib.dates import DateFormatter
 
-        xlabel = self.name.capitalize() + f" [{self.time_format}]"
+        xlabel = PLOT_AXIS_LABEL[self.name] + f" [{self.time_format}]"
```

Comment:
If the name is not in `PLOT_AXIS_LABEL`, it should use `self.name`. After all, `Maps` are sufficiently flexible and we should not prevent plotting for random axis names

Reply:
Yes, I'd use `PLOT_AXIS_LABEL.get(self.name, self.name.capitalize())` to prevent it fails for unknown / uncommon axes.

Reply:
Good point. I agree on this.